includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -310,15 +310,6 @@ public function register_routes() {
 				'permission_callback' => [ $this, 'check_permission' ],
 			]
 		);
-		register_rest_route(
-			$this->namespace,
-			'/' . $this->rest_base . '/request-capability',
```

Comment:
I've noticed this endpoint is now redundant. A bit of history"

- Added here 281c2a00eac31e9a041f90721eaaddb9e0548790 to have JCB request capability notice. 
- Most of the code in the commit above was removed in dcca5d85ebce9b8b4d25b8ab6fabafc6ef82ec85 and 05109176c7b3ead20286f5b9fe1a0b592fdc712a